presto-verifier/src/main/java/io/prestosql/verifier/VerifierConfig.java

```diff
@@ -271,40 +271,42 @@ public VerifierConfig setQueryDatabase(String queryDatabase)
     }
 
     @NotNull
-    public Set<String> getBlacklist()
+    public Set<String> getBannedQueries()
     {
-        return blacklist;
+        return bannedQueries;
     }
 
-    @ConfigDescription("Names of queries which are blacklisted")
-    @Config("blacklist")
-    public VerifierConfig setBlacklist(String blacklist)
+    @ConfigDescription("Names of queries which are banned")
+    @Config("banned-queries")
+    @LegacyConfig("blacklist")
+    public VerifierConfig setBannedQueries(String bannedQueries)
     {
-        ImmutableSet.Builder<String> blacklistBuilder = ImmutableSet.builder();
-        for (String value : Splitter.on(',').trimResults().omitEmptyStrings().split(blacklist)) {
-            blacklistBuilder.add(value);
+        ImmutableSet.Builder<String> bannedBuilder = ImmutableSet.builder();
+        for (String value : Splitter.on(',').trimResults().omitEmptyStrings().split(bannedQueries)) {
+            bannedBuilder.add(value);
         }
 
-        this.blacklist = blacklistBuilder.build();
+        this.bannedQueries = bannedBuilder.build();
         return this;
     }
 
     @NotNull
-    public Set<String> getWhitelist()
+    public Set<String> getAllowQueries()
     {
-        return whitelist;
+        return allowedQueries;
     }
 
-    @ConfigDescription("Names of queries which are whitelisted. Whitelist is applied before the blacklist")
-    @Config("whitelist")
-    public VerifierConfig setWhitelist(String whitelist)
+    @ConfigDescription("Names of queries which are allowed. If non-empty, only allowed queries are used.")
+    @Config("allowList")
```

Comment:
Let's call this `allowed-queries`

Make sure to update `TestVerifierConfig` to match

Reply:
✅